test/over_react_redux/connect_flux_integration_test.dart

```diff
@@ -0,0 +1,347 @@
+        )(ConnectFluxCounter);
+
+        final ConnectedReduxComponent = connect<FluxStore, CounterProps>(
+          mapStateToProps: (state) => (Counter()..currentCount = state.count),
+          mapDispatchToProps: (dispatch) =>
+              (Counter()..increment = () => dispatch(IncrementAction())),
+        )(Counter);
+
+        final jacket = mount((ReduxProvider()..store = store1)(
+          (ConnectedFluxComponent()..addTestId('flux-component'))(),
+          (ConnectedReduxComponent()..addTestId('redux-component'))(),
+        ));
+
+        final fluxCounter = queryByTestId(jacket.mountNode, 'flux-component');
+        final reduxCounter = queryByTestId(jacket.mountNode, 'redux-component');
+
+        final fluxButton = queryByTestId(fluxCounter, 'button-increment');
+        final reduxButton = queryByTestId(reduxCounter, 'button-increment');
+
+        click(fluxButton);
+        await Future(() {});
+
+        expect(findDomNode(fluxCounter).innerHtml, contains('Count: 1'));
+        expect(findDomNode(reduxCounter).innerHtml, contains('Count: 0'));
+
+        store1.dispatch(ResetAction());
+        await Future(() {});
+
+        expect(findDomNode(fluxCounter).innerHtml, contains('Count: 0'));
+        expect(findDomNode(reduxCounter).innerHtml, contains('Count: 0'));
+
+        click(reduxButton);
+        await Future(() {});
+
+        expect(findDomNode(fluxCounter).innerHtml, contains('Count: 1'));
+        expect(findDomNode(reduxCounter).innerHtml, contains('Count: 0'));
+      });
+
+      test('properly maps store instances to actions', () {
+        FluxActions tempActions = FluxActions();
+        final tempFluxStore = FluxStore(tempActions);
+        final tempFluxStore2 = FluxStore(tempActions);
+
+        // Unused, but required to set up the actions
+        // ignore: unused_local_variable
+        final tempAdapterStore =
+            FluxToReduxAdapterStore(tempFluxStore, tempActions);
+        // ignore: unused_local_variable
+        final tempAdapterStore2 =
+            FluxToReduxAdapterStore(tempFluxStore2, tempActions);
+
+        expect(actionsForStore[tempFluxStore], tempActions);
+        expect(actionsForStore[tempFluxStore2], tempActions);
+        expect(actionsForStore[tempFluxStore2], actionsForStore[tempFluxStore]);
+      });
+    });
+
+    group('ConnectFluxAdapterStore', () {
+      UiFactory<ConnectFluxCounterProps> ConnectedFluxComponent;
+      UiFactory<CounterProps> ConnectedReduxComponent;
+      TestJacket jacket;
+      Element connectFluxCounter;
+      Element reduxCounter;
+      Element fluxCounter;
+      List<Element> containerList;
+
+      void verifyCount(List<Element> containers, int count) {
+        for (final container in containers) {
+          expect(findDomNode(container).innerHtml, contains('Count: $count'));
+        }
+      }
+
+      Future<Duration> waitForUpdate() async {
+        return Future.delayed(Duration(seconds: 1));
```

Comment:
This is a pretty long time to wait in unit tests. Flux components are delayed by animation frames, so can we use that instead?
```dart
await window.animationFrame;
await window.animationFrame;
```

Reply:
Yep, much better!